packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -356,6 +369,19 @@ class VMService {
     }
   }
 
+  Future<Map<String, dynamic>> _handleLoadDartLibrarySources(Map<String,
+      dynamic> data) async {
+    final path = data['kernelPath'];
+    final bytes = await fs.file(path).readAsBytes();
+    final result = Base64Encoder().convert(bytes);
```

Comment:
Can you use the top-level dart:convert `base64` variable here (instead of creating a new instance)?